src/components/docs.css

```diff
@@ -10,16 +10,16 @@
 }
 
 .docs-title {
-  color: var(--orange); /* 🔥 flame orange */
+  color: var(--primary); /* 🔥 flame orange */
```

Comment:
This is a good change. You don't have to for this PR. But, removing this comment will help as well.